test/unit/services/urlUtils.ts

```diff
@@ -130,23 +130,25 @@ suite('URL Service', ({ expect, spy, stub }) => {
     });
 
     describe('searchStateToRequest()', () => {
+      let collection = 'a collection';
+      let page = 324;
+      let pageSize = 25;
+      let query = 'dress';
+      let refinements = [1,2,3,4,5];
+      let skip = 30;
+      let sort = { a: 'b' };
```

Comment:
You should declare these here and then assign them in a `beforeEach` to make sure that your tests are isolated. If they don't change, you can make them `const` instead.

Reply:
Noted, will do.